cg/models/cg_config.py

```diff
@@ -257,6 +257,7 @@ class CGConfig(BaseModel):
     environment: Literal["production", "stage"] = "stage"
     flow_cells_dir: str
     madeline_exe: str
+    nanopore_data_directory: str
```

Comment:
```suggestion
    nanopore_flow_cells_dir: str
```

The code above makes the distinction between `illumina_flow_cells` and `nanopore_flow_cells` Maybe we can keep that theme here as well. It would also give a better context than `nanopore_data`

Reply:
So the structure is like this `{experiment_id}/{sample_id}/{start_time}_{device_ID}_{flow_cell_id}_{short_protocol_run_id}` where the flow cell is three levels down. I thought calling it flow_cell directory could be misleading.

Reply:
Will leave this as is, and revise the naming after the folder restructure